### PR TITLE
HOCS-3028 : CCT Triage escalate to WFM

### PR DESCRIPTION
**Pull request by MarkWrightBjss**
Renamed "Response Escalate" to just "Escalate", which is now also used for Triage escalation.
Merge to epic.

### CODE REVIEW

File: src/main/resources/processes/COMP_SERVICE_DRAFT.bpmn [hunk 1]
```diff
@@ -52,29 +54,105 @@
...
+    <bpmn:sequenceFlow id="SequenceFlow_0wewq10" sourceRef="ExclusiveGateway_0yphhsk" targetRef="Service_UpdateTeamByStageAndTexts_Send">
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${CctDraftResult == "Send"}</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
-    <bpmn:serviceTask id="ServiceTask_UpdateTeamByStageAndTexts_Resp_Escalate" name="Update Team for Service Response Escalate" camunda:expression="${bpmnService.updateTeamByStageAndTexts(execution.getVariable(&#34;CaseUUID&#34;),execution.processBusinessKey,&#34;COMP_SERVICE_RESP_ESCALATE&#34;,&#34;QueueTeamUUID&#34;,&#34;QueueTeamName&#34;,&#34;Stage&#34;)}">
-      <bpmn:incoming>SequenceFlow_03k2r1l</bpmn:incoming>
+    <bpmn:serviceTask id="Service_UpdateTeamByStageAndTexts_Resp_Escalate" name="Update Team for Service Escalate" camunda:expression="${bpmnService.updateTeamByStageAndTexts(execution.getVariable(&#34;CaseUUID&#34;),execution.processBusinessKey,&#34;COMP_SERVICE_ESCALATE&#34;,&#34;QueueTeamUUID&#34;,&#34;QueueTeamName&#34;,&#34;Stage&#34;)}">
```
Comment (robertdeniszczyc2) [inline src/main/resources/processes/COMP_SERVICE_DRAFT.bpmn:64]: Should this still be called ..._Resp_Escalate as we have renamed elsewhere to Service Escalate?

File: src/main/resources/processes/COMP_SERVICE_DRAFT.bpmn [hunk 2]
```diff
@@ -52,29 +54,105 @@
...
     </bpmn:serviceTask>
-    <bpmn:sequenceFlow id="SequenceFlow_03k2r1l" sourceRef="ExclusiveGateway_0yphhsk" targetRef="ServiceTask_UpdateTeamByStageAndTexts_Resp_Escalate">
+    <bpmn:sequenceFlow id="SequenceFlow_03k2r1l" sourceRef="ExclusiveGateway_0yphhsk" targetRef="Screen_Escalate">
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${CctDraftResult == "Escalate"}</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
-    <bpmn:sequenceFlow id="SequenceFlow_05snos3" sourceRef="ServiceTask_UpdateTeamByStageAndTexts_Resp_Escalate" targetRef="EndEvent_COMP_SERVICE_DRAFT" />
-    <bpmn:serviceTask id="ServiceTask_UpdateTeamByStageAndTexts_Draft" name="Update Team for Service Triage" camunda:expression="${bpmnService.updateTeamByStageAndTexts(execution.getVariable(&#34;CaseUUID&#34;),execution.processBusinessKey,&#34;COMP_SERVICE_TRIAGE&#34;,&#34;QueueTeamUUID&#34;,&#34;QueueTeamName&#34;,&#34;Stage&#34;)}">
-      <bpmn:incoming>SequenceFlow_0v6jrro</bpmn:incoming>
+    <bpmn:sequenceFlow id="SequenceFlow_05snos3" sourceRef="Service_UpdateTeamByStageAndTexts_Resp_Escalate" targetRef="EndEvent_COMP_SERVICE_DRAFT" />
```
Comment (robertdeniszczyc2) [inline src/main/resources/processes/COMP_SERVICE_DRAFT.bpmn:71]: As above

File: src/main/resources/processes/COMP_SERVICE_DRAFT.bpmn (hunk 1 above)
Comment (MarkWrightBjss) [reply src/main/resources/processes/COMP_SERVICE_DRAFT.bpmn]: Fixed, thank you.

File: src/main/resources/processes/COMP_SERVICE_DRAFT.bpmn (hunk 2 above)
Comment (MarkWrightBjss) [reply src/main/resources/processes/COMP_SERVICE_DRAFT.bpmn]: As above

**Pull request merged**